compiler/luci/pass/src/CircleOptimizer.cpp

```diff
@@ -209,6 +210,9 @@ void CircleOptimizer::quantize(loco::Graph *g) const
     luci::QuantizeWithMinMaxPass quantizer(str_to_dtype(input_dtype), str_to_dtype(output_dtype),
                                            str_to_granularity(granularity));
     quantizer.run(g);
+
+    luci::PropagateConcatenationQparamPass propagator;
+    propagator.run(g);
```

Comment:
There is no solid reason for that.

I made it as a Pass simply for the consistency with other optimizations. It would be ok to change it to a function if there is a clear benefit.

Reply:
The benefit is - less dependencies and less code to support. Never mind, I was just curious.